appng-tools/src/test/java/org/appng/tools/os/OperatingSystemTest.java

```diff
@@ -0,0 +1,50 @@
+package org.appng.tools.os;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class OperatingSystemTest {
+
+	@Test
+	public void testOsDetection() {
+		Assert.assertEquals(OperatingSystem.LINUX, OperatingSystem.detect("Debian GNU/Linux Stretch"));
```

Comment:
Easier: Use `Assert.assertFalse()` and `Assert.assertTrue()` instead.

Reply:
Thanks, pull request has been amended.